ebusd-0.5.x/ochsner/ote2.csv

```diff
@@ -0,0 +1,43 @@
+*r,ochsner,,,,15,0621,,,,,,,,,,,,,
+*w,ochsner,,,,15,0621,,,,,,,,,,,,,
+r,temperature,outside,Au�entemperatur,,,,00800040,,,parameter,,,,,,temperature,,,,
+r,temperature,water,Warmwasser Temperatur,,,,00840040,,,parameter,,,,,,temperature,,,,
+r,temperature,buffer,Vorlauftemperatur,,,,00e00040,,,parameter,,,,,,temperature,,,,
+r,heating,status,,,,,77800008,,,parameter,,,,status,,UCH,,,,
```

Comment:
the trick is in the first line of my suggestion: there is an extra "IGN:2" field added to the defaults line, which will automatically be placed before the individual fields of the actual definition. i.e. the suffix you don't know the meaning of yet, is simply ignored.